src/gui/accountstate.h

```diff
@@ -161,6 +162,32 @@ class AccountState : public QObject, public QSharedData
     ///Asks for user credentials
     void handleInvalidCredentials();
 
+    /** Returns the user status (Online, Dnd, Away, Offline, Invisible)
+     *  https://gist.github.com/georgehrke/55a0412007f13be1551d1f9436a39675
+    */
+    UserStatus::Status status() const;
+
+    /** Returns the user status Message (emoji + text)
+    */
+    QString statusMessage() const;
+
+    /** Returns the user status icon url
+    */
+    QUrl statusIcon() const;
+
+    /** Returns the notifications status retrieved by the notificatons endpoint
+     *  https://github.com/nextcloud/desktop/issues/2318#issuecomment-680698429
+    */
+    bool isDesktopNotificationsAllowed() const;
+
+    /** Set desktop notifications status retrieved by the notificatons endpoint
+    */
+    void setDesktopNotificationsAllowed(const bool isAllowed);
```

Comment:
Please avoid const for base type parameters.

Mostly a question of consistency really.

Reply:
@er-vin I always wondered about this, if it makes sense to make primitive (base) types `const`. In theory, you could modify `isAllowed` in the function if it is not const so there would be some benefit if one makes them `const`.